### PR TITLE
Add otpmml

### PR DESCRIPTION
**Pull request by jschueller**

### CODE REVIEW

File: recipes/otpmml/bld.bat [hunk 1]
```diff
@@ -0,0 +1,2 @@
+if "%ARCH%"=="32" (set CPU_ARCH=i686) else (set CPU_ARCH=x86_64)
+%PKG_NAME%-%PKG_VERSION%-py%PY_VER%-%CPU_ARCH%.exe /userlevel=1 /S /FORCE /D=%PREFIX%
```
Comment (jakirkham) [inline recipes/otpmml/bld.bat]: Can we not build this on Windows too? Seems like we are using `cmake`, which should make this very easy.

Comment (jschueller) [inline recipes/otpmml/bld.bat]: @jakirkham the problem is building openturns with msvc; we only succeeded on building for vc10, without bindings. Also the build time would be a problem.

**Comment by jschueller** [comment]
@ocefpaf, and this one ?

**Comment by ocefpaf** [comment]
> @ocefpaf, and this one ?

LGTM. Let's just wait for `Travis-CI` to process the previous 2 merges.

**Comment by jschueller** [comment]
@ocefpaf thanks!

**Comment by ocefpaf** [comment]
> @ocefpaf thanks!

No problem. Due to the high level of noise in staged-recipes feel free to ping me on things that are stuck. (But I might not see it the first time :smile:)

**Pull request merged**